phpBB/phpbb/db/migration/tool/permission.php

```diff
@@ -704,9 +704,6 @@ public function reverse()
 			break;
 		}
 
-		if ($call)
-		{
-			return call_user_func_array(array(&$this, $call), $arguments);
-		}
+		return $call ? call_user_func_array(array(&$this, $call), $arguments) : null;
 	}
 }
```

Comment:
permission_set method should return void in annotations
Also I would change
```php
return $this->permission_set($role_name, $auth_option, 'role', $has_permission);
```

to

```php
$this->permission_set($role_name, $auth_option, 'role', $has_permission);
return;
```
to make it more understandable